pkgs/development/tools/build-managers/gradle/default.nix

```diff
@@ -294,21 +293,17 @@ rec {
                     nativeBuildInputs = [
                       (gradle.override {
                         javaToolchains = [
-                          jdk11
+                          # this version has to match what is configured in tests/toolchains/build.gradle
```

Comment:
Would it be possible to snag the version from the toolchain and substitute it into `build.gradle` (using [`substituteAllInPlace`](https://nixos.org/manual/nixpkgs/unstable/#fun-substituteAllInPlace) perhaps) instead of this requirement?